contribs/gnomd/main.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	markdown "github.com/MichaelMure/go-term-markdown"
+)
+
+func main() {
+	// If no arguments are provided, read from stdin
+	if len(os.Args) == 1 {
```

Comment:
```suggestion
	if len(os.Args) <= 1 {
```

Just a good habit. Simply because passing 0 arguments to a command is possible, and has been used before for exploits.